packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -251,7 +256,7 @@ export class ExtensionInstance<TConfiguration extends BaseConfigType = BaseConfi
     }
 
     if (!configValue) return undefined
-    return {uuid: identifiers.extensions[this.localIdentifier]!, config: JSON.stringify(configValue), context: ''}
+    return {uuid: identifiers.extensions[this.localIdentifier]!, config: configValue, context: ''}
```

Comment:
This function is now used by 'dev' as well, where it needs to add to the config before stringifying. Consumers are now responsible for their own `stringify`.

Reply:
Much better, thanks!